app/models/crawl_result.rb

```diff
@@ -6,6 +6,7 @@
 #  keyword       :string           not null
 #  search_time   :float            default(0.0), not null
 #  source        :text
+#  status        :integer          default("pending"), not null
```

Comment:
Postgres and Rails also support enum as a column type. So that could eb used here 💭